OsmAnd/src/net/osmand/plus/views/layers/MapSelectionHelper.java

```diff
@@ -324,7 +324,16 @@ private void selectObjectsFromOpenGl(@NonNull MapSelectionResult result, @NonNul
 								}
 								if (onPathMapSymbol == null) {
 									amenity = getAmenity(result.objectLatLon, obfMapObject);
-									if (amenity == null) {
+									if (amenity != null) {
+										RasterMapSymbol rasterMapSymbol = null;
+										try {
+											rasterMapSymbol = RasterMapSymbol.dynamic_pointer_cast(symbolInfo.getMapSymbol());
+										} catch (Exception ignore) {
```

Comment:
log.error or log.warn is must have

Reply:
No need log.error here. It is just trying the cast value (analog instanceof in Java)